src/libraries/Microsoft.Extensions.FileSystemGlobbing/src/FilePatternMatch.cs

```diff
@@ -66,13 +66,10 @@ public override bool Equals(object obj)
         /// Gets a hash for the file pattern match.
         /// </summary>
         /// <returns>Some number</returns>
-        public override int GetHashCode()
-        {
-            var hashCodeCombiner = HashCodeCombiner.Start();
-            hashCodeCombiner.Add(Path, StringComparer.OrdinalIgnoreCase);
-            hashCodeCombiner.Add(Stem, StringComparer.OrdinalIgnoreCase);
+        public override int GetHashCode() =>
```

Comment:
Do we have good test coverage for edge-cases of Equals/GetHashCode to make sure they agree?  Not seeing anything that stands out as buggy here but want to understand how much we should scrutinize.

Reply:
Same question for StringValues below.

Reply:
StringValues has pretty good test coverage for GetHashCode, even edge-cases:

https://github.com/dotnet/runtime/blob/6072e4d3a7a2a1493f514cdf4be75a3d56580e84/src/libraries/Microsoft.Extensions.Primitives/tests/StringValuesTests.cs#L588-L612

https://github.com/dotnet/runtime/blob/6072e4d3a7a2a1493f514cdf4be75a3d56580e84/src/libraries/Microsoft.Extensions.Primitives/tests/StringValuesTests.cs#L93-L117

I didn't see any real tests for FilePatternMatch.GetHashCode, so I added a couple.